lib/linguist/languages.yml

```diff
@@ -1339,6 +1339,7 @@ GDB:
   language_id: 122
 GDScript:
   type: programming
+  color: "#478cbf"
```

Comment:
The hex color here isn't visually distinct enough from other languages already supported by linguist. If these are too close, they make the repository breakdown graph at the top of your repo on github confusing.

You can either revert this commit, or you'll need to update this color to something that doesn't conflict with existing supported languages.